grakn/rpc/session.py

```diff
@@ -47,7 +50,10 @@ def __init__(self, client, database: str, session_type: SessionType, options=Gra
 
         self._session_id = self._grpc_stub.session_open(open_req).session_id
         self._is_open = True
-        self._pulse = self._scheduler.enter(5, 1, self._transmit_pulse, ())
+        self._scheduler.enter(5, 1, self._transmit_pulse, ())
```

Comment:
@alexjpwalker `5` and `1` look awfully like magic numbers here :slightly_smiling_face: the way to improve it would be to name arguments (so we don't need to look into docs to understand them), separate values into constants (to avoid magic numbers) and avoid passing optional arguments (`argument` is only optional since 3.3 so it really depends on minimum Python we support)

```suggestion
        self._scheduler.enter(delay=SESSION_PULSE_DELAY, priority=SESSION_PULSE_PRIORITY, action=self._transmit_pulse)
```

Reply:
Updated to `self._scheduler.enter(delay=self._PULSE_FREQUENCY_SECONDS, priority=1, action=self._transmit_pulse, argument=())`. (We probably won't officially support older Pythons but I see no reason to _intentionally_ break compatibility)